shared/testutil/block.go

```diff
@@ -80,6 +81,10 @@ func NewBeaconBlockAltair() *ethpb.SignedBeaconBlockAltair {
 				Deposits:          []*ethpb.Deposit{},
 				ProposerSlashings: []*ethpb.ProposerSlashing{},
 				VoluntaryExits:    []*ethpb.SignedVoluntaryExit{},
+				SyncAggregate: &ethpb.SyncAggregate{
+					SyncCommitteeBits:      make([]byte, len(bitfield.NewBitvector512())),
```

Comment:
This seems weird to allocate twice the data. Can you just use `bitfield.NewBitvector512()`?
```suggestion
					SyncCommitteeBits:      bitfield.NewBitvector512(),
```

I guess it's just for tests, but could develop into an anti-pattern if someone copied it into a production codepath.

Reply:
Surprisingly that didn't work. Got:
```
shared/testutil/block.go:85:6: cannot use bitfield.NewBitvector512() (type bitfield.Bitvector512) as type bitfield.Bitvector32 in field value
```

Trying `make([]byte, 64)`